content/plugins/aria.md

```diff
@@ -91,3 +85,64 @@ Then the `cloudify.aria.nodes.Service` node containing the CSAR with the above s
 
 
 # Examples
+
+In this section we will run the ARIA hello-world example (which is written in 
+TOSCA) via Cloudify, using the Cloudify ARIA Plugin.
+
+In a nutshell, the ARIA hello world example creates a web server on the machine
+ARIA is installed on, and exposes port 9090 to display a 'hello world'
```

Comment:
listens, not exposes. If the port is locked on some level, it will remain locked...